core/src/main/resources/webroot/swagger-ui/absolute-path.js

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
Instead of copying swagger-ui is it possible to use a webjar (https://www.webjars.org/) instead so we have easy versioning and don't need to copy so many resource files into the project?

Reply:
Hmm ... I can give it a go ... it would mean we would have to enable VertX CP loading.

Reply:
Instead of using webjars (which doesn't allow to load our custom openapi yaml), use unpkg interface as suggested by official swagger-ui installation.